java/src/main/java/genericpubsub/Publish.java

```diff
@@ -16,7 +16,7 @@
 import utility.ExampleConfigurations;
 
 /**
- * A single-topic publisher that creates a CarMaintenance event and publishes it. This example uses
+ * A single-topic publisher that creates an Order_Event event and publishes it. This example uses
```

Comment:
Again a comment about the label vs name of the event. This applies to all the occurrences in other files.